plugins/spark/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ cmake_minimum_required(VERSION 3.3)
 
 project(spark-integration)
 
+option(DOWNLOAD_SPARK_JARS "Download spark jars if jars aren't supplied on command line" OFF)
```

Comment:
let's flip this logic, so that we submodule gold versions of the jars, and allow users to overwrite with local (or remote) SNAPSHOTS...